crypto-ffi/src/core_crypto/conversation.rs

```diff
@@ -24,7 +24,7 @@ impl CoreCryptoFfi {
     pub async fn conversation_epoch(&self, conversation_id: &ConversationId) -> CoreCryptoResult<u64> {
         let conversation = self
             .inner
-            .get_raw_conversation(conversation_id)
+            .get_raw_conversation(&conversation_id.as_ref().into())
```

Comment:
So we have a borowed `ConversationId`, and we manually take a reference to get a byte slice, then call `.into()` to get a vector, then borrow that? _Why_?

It really looks like the API of `get_raw_conversation` should be adjusted.

Reply:
get_raw_conversation lives in `core_crypto` and can't import the ConversationId type of `crypto-ffi`. How can I avoid touching the caller side. 
If we separate the ConversationId byteswrapper type in ffi from the ConversationId core_crypto type I don't know how adjusting the signature helps. At some point this translation has to happen in crypto_ffi.